### ISSUE TITLE
SpinButton: [keros bug]: aria-label attribute does not exist or is empty, when passing aria label through spinbutton prop

### ISSUE THREAD

**Issue by v-pekuma** [OfficeDev/office-ui-fabric-react#7345]
### Environment Information 

- __Package version(s)__: 6.46.0
- __Browser and OS versions__: chrome and windows 10

### Please provide a reproduction of the bug in a codepen:
 https://codepen.io/anon/pen/roNQLg
 
#### Actual behavior:
 
we have a keros issue for spin button control: aria-label attribute does not exist or is empty.
By adding aria label to SpinButton, aria label added to outer div of the input, as given in screenshot below:

![spinbutton aria label](https://user-images.githubusercontent.com/45751963/49721499-58c72c80-fc88-11e8-92c7-f9f3e08709a1.JPG)

 
#### Expected behavior:
 
Aria label property should be added on input control instead of outer div. So that keros do not show any error.

**Comment by khmakoto** [comment]
@v-pekuma After some discussions we think there might be some implications with this change given the existing `SpinButton` code.

Paraphrasing what @KevinTCoughlin mentions in #7352, `SpinButton` has a required prop `label` which is _always_ rendered as an element referred to by `aria-labelledby`.

Apparently, `aria-labelledby` takes precedence over other ARIA attributes such as `aria-label`.

>Importantly, aria-labelledby overrides all other name sources for an element. So, for example, if an element has both an aria-labelledby and an aria-label, or an aria-labelledby and a native HTML label, the aria-labelledby label always takes precedence.

Given that, in the codepen above, the `aria-labelledby` DOM node has an empty string, perhaps the fix is on the implementation side to provide the _required_ `label` prop. The following is a [codepen](https://codepen.io/khmakoto/pen/yGYbGb) doing this.

If this doesn't fix the issue, then it is possible that additional code changes are needed so that `aria-label` is not overridden by `aria-labelledby` on the same element, so please let us know if this fixes the issue.
